docs/examples/app-the-new-thing/interbit.config.js

```diff
@@ -0,0 +1,82 @@
+
+const config = {
+  // TODO: Ensure that at least one account chain is set as a peer
+  peers: ['localhost:5025'],
+  adminValidators: [PUBLIC_KEY],
+  staticChains: {
+    // The public chain runs on the browser and is the entry point for the application
+    [chainAliases.PUBLIC]: {
+      covenant: 'template-public',
+      config: {
+        validators: [PUBLIC_KEY],
+        joins: {
+          consume: [
+            {
+              alias: chainAliases.CONTROL,
+              path: ['interbitServices'],
+              joinName: 'INTERBIT_SERVICES'
+            },
+            {
+              alias: chainAliases.CONTROL,
+              path: ['privateChainHosting'],
+              joinName: 'HOSTING_SPONSOR'
+            }
+          ]
+        }
+      }
+    },
+    // The control chain may contain sensitive application configuration. It runs
+    // on validator nodes, but not on the browser.
+    [chainAliases.CONTROL]: {
+      applyInterbuffer: true,
+      covenant: 'template-control',
+      config: {
+        validators: [PUBLIC_KEY],
+        joins: {
+          provide: [
+            {
+              alias: chainAliases.PUBLIC,
+              path: ['interbitServices', 'shared'],
+              joinName: 'INTERBIT_SERVICES'
+            },
+            {
+              alias: chainAliases.PUBLIC,
+              path: ['privateChainHosting', 'shared'],
+              joinName: 'HOSTING_SPONSOR'
+            }
+          ]
+        }
+      }
+    }
+    // Private chains are created dynamically and is not part of the initial configuration
+  },
+  covenants: {
+    // All covenants used by the application, including covenants for
+    // dynamically created chains
+    'template-public': {
+      location: path.join(__dirname, 'src/interbit/public')
+    },
+    'template-control': {
+      location: path.join(__dirname, 'src/interbit/control')
+    },
+    'template-private': {
+      location: path.join(__dirname, 'src/interbit/private')
+    }
+  },
+  apps: {
+    theNewThing: {
+      peers: ['localhost:5000'], // the peers the browser should connect to
+      chains: [chainAliases.PUBLIC, chainAliases.CONTROL], // the chains that need to load in the browser
+      appChain: chainAliases.PUBLIC, // The chain that the static page is loaded on
+      indexLocation: path.join(__dirname, 'public/index.html'), // the index.html to update with the app info
```

Comment:
I haven't seen the chainAliases file yet. I assume/hope you have changed the aliases so they won't conflict with template. Please and thank you.